Flaky test test_softmin failed CI

test_operator.test_softmin failed the CI at http://jenkins.mxnet-ci.amazon-ml.com/blue/organizations/jenkins/mxnet-validation%2Fwindows-gpu/detail/PR-13364/8/pipeline on stage "Python 3: MKLDNN-GPU Win" on unrelated PR #13364. 

See output below:
```
======================================================================

FAIL: test_operator.test_softmin

----------------------------------------------------------------------

Traceback (most recent call last):

  File "C:\Anaconda3\envs\py3\lib\site-packages\nose\case.py", line 197, in runTest

    self.test(*self.arg)

  File "C:\jenkins_slave\workspace\ut-python-gpu\tests\python\unittest\common.py", line 173, in test_new

    orig_test(*args, **kwargs)

  File "C:\jenkins_slave\workspace\ut-python-gpu\tests\python\unittest\test_operator.py", line 4408, in test_softmin

    rtol=rtol, atol=atol, grad_req=req, dtype=dtype)

  File "C:\jenkins_slave\workspace\ut-python-gpu\windows_package\python\mxnet\test_utils.py", line 1131, in check_symbolic_backward

    equal_nan=equal_nan)

  File "C:\jenkins_slave\workspace\ut-python-gpu\windows_package\python\mxnet\test_utils.py", line 495, in assert_almost_equal

    raise AssertionError(msg)

AssertionError: 

Items are not equal:

Error 1.119420 exceeds tolerance rtol=0.010000, atol=0.005000.  Location of maximum error:(0, 1, 0, 0), a=0.000000, b=-0.005660

 EXPECTED_softmin9_data: array([[[[ 0.,  0.,  0.],

         [ 0.,  0.,  0.]],

...

 BACKWARD_softmin9_data: array([[[[ 0.00006208, -0.00026137,  0.00015508],

         [ 0.00023366,  0.00007945,  0.00000728]],

...

-------------------- >> begin captured logging << --------------------

common: INFO: Setting test np/mx/python random seeds, use MXNET_TEST_SEED=350762988 to reproduce.

--------------------- >> end captured logging << ---------------------
```

--- a/tests/python/unittest/test_operator.py
+++ b/tests/python/unittest/test_operator.py
@@ -3384,7 +3384,6 @@ def check_l2_normalization(in_shape, mode, dtype, norm_eps=1e-10):
 
 
 @with_seed()
-@unittest.skip("Flaky test: https://github.com/apache/incubator-mxnet/issues/15004")
 def test_l2_normalization():
     for dtype in ['float16', 'float32', 'float64']:
         for mode in ['channel', 'spatial', 'instance']:
@@ -4859,7 +4858,6 @@ def test_where():
     test_1d_cond()
 
 
-@unittest.skip("Flaky test. Tracked in https://github.com/apache/incubator-mxnet/issues/13600")
 @with_seed()
 def test_softmin():
     for ndim in range(1, 5):

